src/integrationTest/java/uk/gov/hmcts/darts/authorisation/controller/AuthorisationControllerIntTest.java

```diff
@@ -36,6 +38,7 @@
 import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
 
 @AutoConfigureMockMvc
+@Transactional
```

Comment:
we have been trying to remove @Transactional from all our tests, so try not to add any.

Reply:
When you run these tests individually without the @Transactional annotation, they pass successfully. However, when you execute the tests together as a suite, you encounter failures related to detached entity errors.